<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoterManagement</class>
 <widget class="QMainWindow" name="VoterManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>582</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1262</width>
    <height>582</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PolliTech</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../VVM.qrc">
    <normaloff>:/backgrounds/Smol logo no bg.png</normaloff>:/backgrounds/Smol logo no bg.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>303</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
background-color: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item alignment="Qt::AlignmentFlag::AlignBottom">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	font: 700 9pt &quot;Segoe UI&quot;;
	font-size: 35px;
	color: #0A1C3A;
	background-color: none;
}</string>
           </property>
           <property name="text">
            <string>MANAGE VOTERS</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextInteractionFlag::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="topMargin">
            <number>5</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <item alignment="Qt::AlignmentFlag::AlignBottom">
                <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
	font: 600 9pt &quot;Segoe UI&quot;;
	font-size: 17px;
	color: #0A1C3A;
	background-color: none;
}</string>
                 </property>
                 <property name="text">
                  <string>FIRST NAME</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_first_name">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::StrongFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
background-color: rgba(255,255,255,200);
color: #0A1C3A;
font: 14pt &quot;Segoe UI&quot;;
font-size: 16px;
padding: 5px;
border: 1px solid #0A1C3A;
}</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <item alignment="Qt::AlignmentFlag::AlignBottom">
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
	font: 600 9pt &quot;Segoe UI&quot;;
	font-size: 17px;
	color: #0A1C3A;
	background-color: none;
}</string>
                 </property>
                 <property name="text">
                  <string>VOTER ID</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_voter_id">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::StrongFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
background-color: rgba(255,255,255,200);
color: #0A1C3A;
font: 14pt &quot;Segoe UI&quot;;
font-size: 16px;
padding: 5px;
border: 1px solid #0A1C3A;
}</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignmentFlag::AlignBottom">
                <widget class="QLabel" name="label_4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
	font: 600 9pt &quot;Segoe UI&quot;;
	font-size: 17px;
	color: #0A1C3A;
	background-color: none;
}</string>
                 </property>
                 <property name="text">
                  <string>LAST NAME</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_last_name">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::StrongFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
background-color: rgba(255,255,255,200);
color: #0A1C3A;
font: 14pt &quot;Segoe UI&quot;;
font-size: 16px;
padding: 5px;
border: 1px solid #0A1C3A;
}</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="leftMargin">
                <number>80</number>
               </property>
               <property name="rightMargin">
                <number>80</number>
               </property>
               <item alignment="Qt::AlignmentFlag::AlignBottom">
                <widget class="QLabel" name="label_5">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
	font: 600 9pt &quot;Segoe UI&quot;;
	font-size: 17px;
	color: #0A1C3A;
	background-color: none;
}</string>
                 </property>
                 <property name="text">
                  <string>AGE</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_age">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::FocusPolicy::StrongFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
background-color: rgba(255,255,255,200);
color: #0A1C3A;
font: 14pt &quot;Segoe UI&quot;;
font-size: 16px;
padding: 5px;
border: 1px solid #0A1C3A;
}</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="topMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	font: italic 11pt &quot;Segoe UI&quot;;
	font-size: 14px;
	color: #0A1C3A;
	background-color: none;
}</string>
             </property>
             <property name="text">
              <string>Voter ID cannot be changed. If incorrect, please delete and re-add the voter</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetFixedSize</enum>
             </property>
             <property name="leftMargin">
              <number>180</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>180</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_back">
               <property name="minimumSize">
                <size>
                 <width>136</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    font: 600 9pt &quot;Segoe UI&quot;;
    font-size: 16px;
    color: white;

    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(1, 120, 200),
        stop:1 rgb(1, 100, 182)
    );

    border: 1px solid rgb(1, 26, 48);
    border-radius: 15px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(30, 145, 220),
        stop:1 rgb(0, 90, 165)
    );
}

QPushButton:pressed {
    background-color: rgb(0, 75, 140);
    padding-top: 6px;
    padding-bottom: 4px;
}

QPushButton:focus {
    outline: none;
    border: 2px solid #0A1C3A;
}</string>
               </property>
               <property name="text">
                <string>BACK</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_insert">
               <property name="minimumSize">
                <size>
                 <width>136</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    font: 600 9pt &quot;Segoe UI&quot;;
    font-size: 16px;
    color: black;
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:0, y2:1,
        stop:0 #6FE1A4, stop:1 #3CA36E
    );
    border: 1px solid #2e7d5d;
    border-radius: 15px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:0, y2:1,
        stop:0 #7FFFB2, stop:1 #4BB97E
    );
}

QPushButton:pressed {
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:0, y2:1,
        stop:0 #3CA36E, stop:1 #6FE1A4
    );
    padding-top: 7px;
    padding-bottom: 3px;
}

QPushButton:focus {
    outline: none;
    border: 2px solid #1A5E3B;
}
</string>
               </property>
               <property name="text">
                <string>INSERT</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="RefreshButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Refresh&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    font: 600 9pt &quot;Segoe UI&quot;;
    font-size: 16px;
    color: black;

    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #B3E5FC,
        stop:1 #4FC3F7
    );

    border: 1px solid #0288D1;
    border-radius: 15px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #E1F5FE,
        stop:1 #81D4FA
    );
}

QPushButton:pressed {
    background-color: #29B6F6;
    padding-top: 6px;
    padding-bottom: 4px;
}

QPushButton:focus {
    outline: none;
    border: 2px solid #0288D1;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../VVM.qrc">
            <normaloff>:/icons/buttons/icons/refresh-page-option.png</normaloff>:/icons/buttons/icons/refresh-page-option.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTableWidget" name="Voter_table">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget {
color: black;
background-color: #ffffff;
border: 1px solid #ccc;
gridline-color: #ddd;
font: 9pt &quot;Segoe UI&quot;;
}
</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::DoubleClicked</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>180</number>
          </property>
          <property name="rightMargin">
           <number>180</number>
          </property>
          <item>
           <widget class="QPushButton" name="ListDeleteButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Delete voter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    font: 600 9pt &quot;Segoe UI&quot;;
    font-size: 16px;
    color: white;

    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(220, 60, 50), 
        stop:1 rgb(180, 30, 30)
    );

    border: 1px solid #5A0A0A; 
    border-radius: 15px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(240, 80, 70),
        stop:1 rgb(200, 40, 40)
    );
}

QPushButton:pressed {
    background-color: rgb(160, 20, 20);
    padding-top: 6px;
    padding-bottom: 4px;
}

QPushButton:focus {
    outline: none;
    border: 2px solid #8B0000;
}
</string>
            </property>
            <property name="text">
             <string>DELETE</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteAll_Button">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Delete all the voters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    font: 600 9pt &quot;Segoe UI&quot;;
    font-size: 16px;
    color: white;

    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(240, 30, 20),
        stop:1 rgb(200, 20, 20)  
    );

    border: 1px solid #8B0000; 
    border-radius: 15px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 rgb(255, 50, 40), 
        stop:1 rgb(210, 40, 30)
    );
}

QPushButton:pressed {
    background-color: rgb(180, 10, 10); 
    padding-top: 6px;
    padding-bottom: 4px;
}

QPushButton:focus {
    outline: none;
    border: 2px solid #B22222; 
}
</string>
            </property>
            <property name="text">
             <string>DELETE ALL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_first_name</tabstop>
  <tabstop>lineEdit_last_name</tabstop>
  <tabstop>lineEdit_voter_id</tabstop>
  <tabstop>lineEdit_age</tabstop>
  <tabstop>pushButton_back</tabstop>
  <tabstop>pushButton_insert</tabstop>
  <tabstop>Voter_table</tabstop>
  <tabstop>ListDeleteButton</tabstop>
  <tabstop>DeleteAll_Button</tabstop>
 </tabstops>
 <resources>
  <include location="../VVM.qrc"/>
 </resources>
 <connections/>
</ui>
